server/chat-plugins/mafia.ts

```diff
@@ -2500,6 +2522,51 @@ export const commands: ChatCommands = {
 			`/mafia spiritstump [player] Kills a player, but allows them to talk during the day, and vote on the lynch.`,
 		],
 
+		'!revealrole': true,
+		revealas: 'revealrole',
+		revealrole(target, room, user, connection, cmd) {
+			const args = target.split(',');
+			let targetRoom = Rooms.get(args[0]);
+			if (!targetRoom || targetRoom.type !== 'chat' || !targetRoom.users[user.id]) {
+			    if (!room || room.type !== 'chat') return this.errorReply(`This command is only meant to be used in chat rooms.`);
+			    targetRoom = room;
+			} else {
+			    args.shift();
+			}
+			const game = targetRoom.getGame(MafiaTracker);
+			if (!game) return user.sendTo(targetRoom, `|error|There is no game of mafia running in this room.`);
+			if (game.hostid !== user.id && !game.cohosts.includes(user.id) && !this.can('mute', null, room)) return;
+			let revealAs = '';
+			let revealedRole = null;
+			if (cmd === 'revealas') {
+				if (!args[0]) {
+					return this.parse('/help mafia revealas');
+				} else {
+					revealedRole = MafiaTracker.parseRole(args.pop()!);
+					const color = MafiaData.alignments[revealedRole.role.alignment].color;
+					revealAs = `<span style="font-weight:bold;color:${color}">${revealedRole.role.safeName}</span>`;
+				}
+			}
+			if (!args[0]) return this.parse('/help mafia revealas');
+			for (const targetUsername of args) {
+				let player = game.playerTable[toID(targetUsername)];
+				if (!player) player = game.dead[toID(targetUsername)];
+				if (player) {
+					game.revealRole(user, player, `${cmd === 'revealas' ? revealAs : player.getRole()}`);
+					game.logAction(user, `revealed ${player.name}`);
+					if (cmd === 'revealas') {
+						game.secretLogAction(user, `$fakerevealed ${player.name} as ${revealedRole!.role.safeName}`);
```

Comment:
```suggestion
						game.secretLogAction(user, `fakerevealed ${player.name} as ${revealedRole!.role.name}`);
```

(`safeName` is just for html)